### DISCUSSION TITLE
Not able to participate in octernship

### DISCUSSION THREAD

**Discussion by abhoygorai** [community/community#65482] [GitHub Education] [answered]
### Hi everyone,

Hey, GitHub community. I am Abhoy Gorai a third-year CSE undergrad. I am confident about my web development skills and I am also good at data structures and algorithms. Still, I am not able to take part in Octernship. It is showing me that I have to contribute a bit more. Is there any possible solution to this problem? 

I really wish to take part and contribute to the projects.

**Comment by humairshoukat** [comment, marked as answer]
As of now GitHub Octernships are only available in 10 countries, you can check those here: https://education.github.com/students/octernships

You're not able to apply because they have a set limit on the number of contributions to apply. I don't exactly what is the limit but what you can do is contribute more to open source projects so that you can apply for octernships in the future.

**Reply by abhoygorai** [reply]
Thanks for you answer
